release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -419,6 +419,12 @@ For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features
 [id="ocp-release-notes-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-18-installation-and-update-gcp-byo-phz_{context}"]
+==== Provide your own private hosted zone when installing a cluster on {gcp-short}
+With this release, you can provide your own private hosted zone when installing a cluster on {gcp-short} into a shared VPC. If you do, you must provision the required API and ingress VIPs in that zone. If you do not provide a private hosted zone, the installation program will provision one automatically.
```

Comment:
> If you do, you must provision the required API and ingress VIPs in that zone. 

Instead, the requirements on the BYO hosted zone are, the zone must be with dns name like `<cluster name>.<base domain>.` and bind to the cluster's VPC network. Nothing else then.